packages/fxa-settings/src/components/Settings/PageSettings/index.tsx

```diff
@@ -31,6 +34,26 @@ export const PageSettings = (_: RouteComponentProps) => {
     GleanMetrics.accountPref.view();
   }, []);
 
+  const accountRecoveryNotificationProps = {
+    headerImage: keyImage,
+    ctaText: ftlMsgResolver.getMsg(
+      'account-recovery-notification-cta',
+      'Create'
+    ),
+    headerValue: ftlMsgResolver.getMsg(
+      'account-recovery-notification-header-value',
+      "Don't lose your data if you forget your password"
+    ),
+    headerDescription: ftlMsgResolver.getMsg(
+      'account-recovery-notification-header-description',
+      'Create an Account Recovery Key to restore your sync browsing data if you ever forget your password.'
+    ),
+    route: '/settings/account_recovery',
+    dismissKey: 'account-recovery-dismissed',
+    metricsPrefix: 'promote-account-recovery',
+    isVisible: !recoveryKey,
```

Comment:
```suggestion
    isVisible: !recoveryKey.exists,
```

Reply:
Assuming we want to display it in Settings now - disregard if we'll want to wait until the follow-up is completed.